### ISSUE TITLE
JavaScript error on download page

### ISSUE THREAD

**Issue by walterdavis** [dequelabs/axe-core#1713]
Expectation: Attempted to analyze the Axe download page on https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/

Actual: Throws this error: 548/t.default@moz-extension://cf7c094d-e109-3b4f-8974-f32702c5645c/background.bundle.js:1:4339

Motivation: This feature seems to work on other pages, but does not work on your page.

<pre><code>
axe-core version: 3.8.0

Browser and Assistive Technology versions

Mac (10.14.5 Mojave) Firefox Quantum 68.0.1

**Comment by WilcoFiers** [comment]
Good catch. Not sure what's up with that. It only seems to happen with the Firefox extension, which sort of suggests Firefox doesn't allow content scripts on extension pages? We'll look at it.

**Comment by roman-II** [comment]
@walterdavis Seems like @WilcoFiers may be onto something, after doing some digging. I found this on a similar issue with React Dev Tools:

> The "dev tools" parts of an extension is not "added to a privileged page", it is "added to a developer toolbox", which may be navigated to or from a privileged page at any time.

> And so, that error message is actually expected, and its main purpose is to make the user aware why the extension has not been able to work as expected (or even create its dev tools panel) on some pages, instead of turning it into a silent error.

A more detailed explanation can be found here: https://bugzilla.mozilla.org/show_bug.cgi?id=1477112

![Screen Shot 2019-07-29 at 2 06 11 PM](https://user-images.githubusercontent.com/4601329/62072783-54e31d00-b20d-11e9-8213-14147ae64390.png)
